cosmo_tester/framework/test_hosts.py

```diff
@@ -193,7 +194,7 @@ def stop(self):
             try:
                 self.run_command('echo Still up...')
                 time.sleep(3)
-            except SSHException:
+            except (SSHException, timeout):
```

Comment:
looking at this line i'm gonna ask "what timeout?!". Especially since like this, it looks like some kinda variable
i think it would be better to do `import socket` and then `except socket.timeout`

Reply:
Makes sense, update incoming.